client/src/main/java/com/mytiki/publish/client/TikiClient.kt

```diff
@@ -321,8 +322,10 @@ object TikiClient {
   fun createLicense(context: Context): CompletableDeferred<Boolean> {
     val isLicenseCreated = CompletableDeferred<Boolean>()
     MainScope().async {
-      license.create(context)
-      isLicenseCreated.complete(license.create(context))
+      val license =
```

Comment:
It should receive an offer and create the license based on the offer. For the OCR we need to create an offer.